system-int-test/test/cmr/system_int_test/access_control/permissions_route_test.clj

```diff
@@ -0,0 +1,44 @@
+(ns cmr.system-int-test.access-control.permissions-route-test
```

Comment:
access-control integration tests are not in `system-int-test`, they are within the access-control project. See `cmr.access-control.int-test.permission-check-test` for existing `get-permissions` tests.

Reply:
Moved